packages/element-app/package.json

```diff
@@ -17,6 +17,8 @@
     "lodash": "^4.17.4",
     "material-auto-rotating-carousel": "^3.0.1",
     "moment": "^2.23.0",
+    "nanobus": "^4.4.0",
```

Comment:
these are dependencies of element-lib, do we really need them here?

Reply:
Storybook errors if we don't add those dependencies:

<img width="719" alt="Screenshot 2019-08-06 at 16 02 18" src="https://user-images.githubusercontent.com/7913565/62546340-99049c00-b863-11e9-8a17-c0055b0a422d.png">

Reply:
ahh, i see... the components that are erroring are for testing those infrastructure components by themselves... probably not a good idea in the long run, since those components are not really useful. I added them as a sanity check when refactoring element-lib, i wanted to be sure things worked in the browser and used storybook.

Reply:
we could remove the infrastructure components from storybook since they are not really reusable